src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -671,9 +674,14 @@ def image_mask_iterator():
             ],
             id=roi_ids,
         )
-
-        image_segmentation.create_plane_segmentation(**plane_segmentation_kwargs)
-
+        plane_segmentation = PlaneSegmentation(**plane_segmentation_kwargs)
+        if include_roi_centroids:
+            tranpose_roi_location_axes = (1, 0) if len(segmentation_extractor.get_image_size()) == 2 else (1, 0, 2)
+            roi_locations = segmentation_extractor.get_roi_locations()[tranpose_roi_location_axes, :].T
```

Comment:
I was kind of confused about the double transpose (as another transpose is repeated on line 680) at first glance. Maybe some comment about that? 

I get that you require to write the images in the expected format (columns vs heights) [hence transpose in lines 677] and then you transpose all the roi_locations to get num_rois in the first dimension. Not related tot his PR but should not we return the roi locations in shape `(num_rois, shape_of_the_images)`?

Reply:
Moved transpose to emphasize connection to writing as column to table